source/fundamentals/indexes.txt

```diff
@@ -61,24 +57,24 @@ from the index, also called a **covered query**.
 
       name_1_age_-1
 
-   MongoDB would use this index when you sort your data by either:
+   MongoDB uses this index when you sort your data by either:
 
    - ``name`` ascending, ``age`` descending
    - ``name`` descending, ``age`` ascending
 
    Specifying a sort order of ``name`` and :guilabel:`age` ascending or :guilabel:`name` and ``age``
-   descending would require an in-memory sort.
+   descending requires an in-memory sort.
 
-For additional information on how to ensure your index covers your query criteria and projection, see the MongoDB manual
+For more information on how to ensure your index covers your query criteria and projection, see the MongoDB manual
 articles on :manual:`query coverage </core/query-optimization/#covered-query>`.
 
 Operational Considerations
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 To improve query performance, build indexes on fields that appear often in your application's queries and operations
-that return sorted results. Each index that you add consumes disk space and memory when active so you should track index
-memory and disk usage for capacity planning. In addition, when a write operation updates an indexed field, MongoDB also
-has to update the related index.
+that return sorted results. Each index that you add consumes disk space and memory when active so you can track index
+memory and disk usage for capacity planning. In addition, when a write operation updates an indexed field, MongoDB
```

Comment:
S: I think changing the should to can changed the meaning of this sentence. Could we say something like

```suggestion
that return sorted results. Each index that you add consumes disk space and memory when active, so we recommend that you track index
memory and disk usage for capacity planning. In addition, when a write operation updates an indexed field, MongoDB
```